src/services/yieldService.ts

```diff
@@ -27,7 +27,7 @@ export default class YieldService {
     this.client = client;
   }
 
-  async getYieldOptions(chainId?: number): Promise<YieldOptions> {
+  async getYieldOptions(chainId?: number, ignoreBlacklist = false): Promise<YieldOptions> {
```

Comment:
Can we rename`ignoreBlacklist` to `useBlacklist` or `listenToBlacklist`? (Here and in all the other files)